tests/system/providers/amazon/aws/example_rds_event.py

```diff
@@ -41,35 +43,6 @@ def create_sns_topic(env_id) -> str:
     return boto3.client('sns').create_topic(Name=f'{env_id}-topic')['TopicArn']
 
 
-@task
-def create_rds_instance(db_name, instance_name) -> None:
```

Comment:
I made the same updates in each system test file

Reply:
Thanks for this!